controllers/admin/AdminCmsContentController.php

```diff
@@ -57,12 +57,16 @@ public function __construct()
                 'icon' => 'icon-trash'
             )
         );
+
+        parent::__construct();
+
         $this->admin_cms_categories = new AdminCmsCategoriesController();
+        $this->admin_cms_categories->tabAccess = $this->tabAccess;
         $this->admin_cms_categories->init();
         $this->admin_cms = new AdminCmsController();
+        $this->admin_cms->tabAccess = $this->tabAccess;
```

Comment:
Because i think there will be no employee which can edit CMS categories but cannot edit CMS pages.